src/util/crypto_hash.h

```diff
@@ -10,10 +10,24 @@ class crypto_hash
 {
 public:
   std::shared_ptr<crypto_hash_private> p_crypto;
+#if BOOST_VERSION >= 108600
```

Comment:
Could you just use a define or a static constant for the array length? 

This way you dont need to repeat this ifdef.

Reply:
done